### PR TITLE
Fix close active document key binding

### LINKED ISSUE #24: ALT GR + W closes the active document
When I use alt gr + w, the current active document is closed, which is not the behavior of common applications(such as google chrome, etc).
I use alt gr + w to create the "?" char.

### PR DESCRIPTION
**Pull request by ceifa**
Fixes #24

### CODE REVIEW

File: src/Tweaks/Editor/CloseActiveDocument.cs [hunk 1]
```diff
@@ -27,7 +27,8 @@ private static void OnKeyUp(object sender, KeyEventArgs e)
         {
             ThreadHelper.ThrowIfNotOnUIThread();
 
-            if ((e.KeyboardDevice.Modifiers & ModifierKeys.Control) == ModifierKeys.Control &&
+            if ((e.KeyboardDevice.Modifiers == ModifierKeys.Control ||
+                e.KeyboardDevice.Modifiers == (ModifierKeys.Control | ModifierKeys.Shift)) &&
```
Comment (madskristensen) [inline src/Tweaks/Editor/CloseActiveDocument.cs:31]: Wouldn't this make it close the document when either Control or Control+Alt (altgr) is pressed?

Comment (ceifa) [reply src/Tweaks/Editor/CloseActiveDocument.cs:31]: No, it will only close if you press Ctrl or Ctrl + Shift.

Comment (madskristensen) [reply src/Tweaks/Editor/CloseActiveDocument.cs:31]: Why Ctrl + Shift?

Comment (ceifa) [reply src/Tweaks/Editor/CloseActiveDocument.cs:31]: Of course. I just tried to bring a more proximity with the chromium behaviour. (Ctrl + Shift + W closes all pages on chromium, but that can be a future feature xD)

Comment (madskristensen) [reply src/Tweaks/Editor/CloseActiveDocument.cs:31]: Oh, I love that idea

**Comment by madskristensen** [comment]
Thanks!!

**Pull request merged**